deploy.go

```diff
@@ -232,14 +237,20 @@ func displayNewCommits(owner string, repo string, c *cli.Context, client *github
 	return nil
 }
 
-func newDeploymentRequest(c *cli.Context) (*github.DeploymentRequest, error) {
-	ref := Ref(c.String("ref"), git.Head)
+var EnvironmentAliases = map[string]string{
```

Comment:
I think this should definitely be configurable since some people may want to create their own aliases, but looks like a good sensible default.